frontend/src/main/components/Jobs/MilkCowsJobForm.js

```diff
@@ -0,0 +1,24 @@
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+function MilkCowsJobForm(submitAction) {
+  const defaultValues = {
+    fail: false,
+    sleepMs: 1000
+};
```

Comment:
We don't need these values for milking the cows; these are only needed for the test job.   We can remove these.